kafka-quarkus-dev/quickstart.yml

```diff
@@ -1,7 +1,7 @@
 apiVersion: console.openshift.io/v1
 kind: QuickStarts
 metadata:
-  name: quarkus
+  name: kafka-quarkus-dev
```

Comment:
No rename here.

Reply:
@pmuir just checking - after our quick sync is it still ok to keep it renamed. 

We can either have:

- kafka-quarkus
- kafka-quarkus-servicebinding

Or 

- kafka-quarkus-dev
- kafka-quarkus-servicebinding